aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -526,6 +532,48 @@ Resources:
 <%  unless load_balancer -%>
           - !ImportValue VPC-ELBSecurityGroup # Accept HTTP traffic on daemon
 <%  end -%>
```

Comment:
These alarms are currently within an `if daemon` conditional block, but we also want these alarms to be created for stacks that don't have `daemon` in their CloudFormation stacks (e.g., where manually-provisioned daemon instances already exist). So we need to insert these newly-added alarm resources _after_ the `<%end-%>` on [line `529`](https://github.com/code-dot-org/code-dot-org/pull/20975/files#diff-8c5f53597b66e42c205fea2a95a1da70L529), so the alarms fall outside that conditional and get created in all stacks.